2024-08-06 19:02:39,371 INFO    StreamThr :2119611 [internal.py:wandb_internal():86] W&B internal server running at pid: 2119611, started at: 2024-08-06 19:02:39.370664
2024-08-06 19:02:39,376 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status
2024-08-06 19:02:39,379 INFO    WriterThread:2119611 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/run-e3axglmp.wandb
2024-08-06 19:02:39,380 DEBUG   SenderThread:2119611 [sender.py:send():382] send: header
2024-08-06 19:02:39,380 DEBUG   SenderThread:2119611 [sender.py:send():382] send: run
2024-08-06 19:02:39,546 INFO    SenderThread:2119611 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files
2024-08-06 19:02:39,546 INFO    SenderThread:2119611 [sender.py:_start_run_threads():1136] run started: e3axglmp with start time 1722942159.370263
2024-08-06 19:02:39,557 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: check_version
2024-08-06 19:02:39,558 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: check_version
2024-08-06 19:02:39,672 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: run_start
2024-08-06 19:02:39,683 DEBUG   HandlerThread:2119611 [system_info.py:__init__():32] System info init
2024-08-06 19:02:39,683 DEBUG   HandlerThread:2119611 [system_info.py:__init__():47] System info init done
2024-08-06 19:02:39,683 INFO    HandlerThread:2119611 [system_monitor.py:start():194] Starting system monitor
2024-08-06 19:02:39,683 INFO    SystemMonitor:2119611 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-06 19:02:39,684 INFO    HandlerThread:2119611 [system_monitor.py:probe():214] Collecting system info
2024-08-06 19:02:39,684 INFO    SystemMonitor:2119611 [interfaces.py:start():190] Started cpu monitoring
2024-08-06 19:02:39,685 INFO    SystemMonitor:2119611 [interfaces.py:start():190] Started disk monitoring
2024-08-06 19:02:39,685 INFO    SystemMonitor:2119611 [interfaces.py:start():190] Started gpu monitoring
2024-08-06 19:02:39,685 INFO    SystemMonitor:2119611 [interfaces.py:start():190] Started memory monitoring
2024-08-06 19:02:39,686 INFO    SystemMonitor:2119611 [interfaces.py:start():190] Started network monitoring
2024-08-06 19:02:39,696 DEBUG   HandlerThread:2119611 [system_info.py:probe():196] Probing system
2024-08-06 19:02:39,699 DEBUG   HandlerThread:2119611 [system_info.py:_probe_git():181] Probing git
2024-08-06 19:02:39,704 DEBUG   HandlerThread:2119611 [system_info.py:_probe_git():189] Probing git done
2024-08-06 19:02:39,704 DEBUG   HandlerThread:2119611 [system_info.py:probe():244] Probing system done
2024-08-06 19:02:39,704 DEBUG   HandlerThread:2119611 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-41-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-06T11:02:39.697111', 'startedAt': '2024-08-06T11:02:39.363773', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/Gemma-2B', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:395000]', '--output_dir', '/home/rwkv/JL/out_model/Gemma-meta-2b', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '16', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': None}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 1732.2135357142854, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 2925.436, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1627.179, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1759.341, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1746.185, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1442.609, 'min': 800.0, 'max': 5600.0}, {'current': 2866.995, 'min': 800.0, 'max': 5600.0}, {'current': 1767.423, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 1582.867, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1780.161, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1703.133, 'min': 800.0, 'max': 4300.0}, {'current': 1346.405, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 443.36705780029297}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.5532341003418}}
2024-08-06 19:02:39,704 INFO    HandlerThread:2119611 [system_monitor.py:probe():224] Finished collecting system info
2024-08-06 19:02:39,704 INFO    HandlerThread:2119611 [system_monitor.py:probe():227] Publishing system info
2024-08-06 19:02:39,704 DEBUG   HandlerThread:2119611 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-06 19:02:39,704 DEBUG   HandlerThread:2119611 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-06 19:02:39,704 DEBUG   HandlerThread:2119611 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-06 19:02:40,546 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/requirements.txt
2024-08-06 19:02:40,546 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/conda-environment.yaml
2024-08-06 19:02:41,389 DEBUG   HandlerThread:2119611 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-06 19:02:41,389 DEBUG   HandlerThread:2119611 [system_info.py:_save_code():90] Saving code
2024-08-06 19:02:41,390 DEBUG   HandlerThread:2119611 [system_info.py:_save_code():111] Saving code done
2024-08-06 19:02:41,390 DEBUG   HandlerThread:2119611 [system_info.py:_save_patches():128] Saving git patches
2024-08-06 19:02:41,393 ERROR   HandlerThread:2119611 [system_info.py:_save_patches():169] Error generating diff: Reference at 'refs/remotes/origin/main' does not exist
2024-08-06 19:02:41,393 DEBUG   HandlerThread:2119611 [system_info.py:_save_patches():170] Saving git patches done
2024-08-06 19:02:41,393 INFO    HandlerThread:2119611 [system_monitor.py:probe():229] Finished publishing system info
2024-08-06 19:02:41,394 DEBUG   SenderThread:2119611 [sender.py:send():382] send: files
2024-08-06 19:02:41,394 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-06 19:02:41,394 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-06 19:02:41,397 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 19:02:41,397 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: stop_status
2024-08-06 19:02:41,442 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 19:02:41,546 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/conda-environment.yaml
2024-08-06 19:02:41,546 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/code/llama/gmm.py
2024-08-06 19:02:41,546 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/wandb-metadata.json
2024-08-06 19:02:41,546 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/code
2024-08-06 19:02:41,546 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/code/llama
2024-08-06 19:02:41,674 DEBUG   SenderThread:2119611 [sender.py:send():382] send: telemetry
2024-08-06 19:02:41,674 DEBUG   SenderThread:2119611 [sender.py:send():382] send: config
2024-08-06 19:02:41,675 DEBUG   SenderThread:2119611 [sender.py:send():382] send: metric
2024-08-06 19:02:41,675 DEBUG   SenderThread:2119611 [sender.py:send():382] send: telemetry
2024-08-06 19:02:41,675 DEBUG   SenderThread:2119611 [sender.py:send():382] send: metric
2024-08-06 19:02:41,675 WARNING SenderThread:2119611 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-06 19:02:41,675 DEBUG   SenderThread:2119611 [sender.py:send():382] send: telemetry
2024-08-06 19:02:41,675 DEBUG   SenderThread:2119611 [sender.py:send():382] send: telemetry
2024-08-06 19:02:41,675 DEBUG   SenderThread:2119611 [sender.py:send():382] send: config
2024-08-06 19:02:41,689 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310
2024-08-06 19:02:41,689 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5179', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:41,689 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E91EC479B1E18D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:02:41,852 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361
2024-08-06 19:02:41,852 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:41,852 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E91EC47F96D6DD</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:02:42,546 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:02:42,800 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310
2024-08-06 19:02:42,800 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5179', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:42,800 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E91EC4BBBBAA05</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:02:43,140 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361
2024-08-06 19:02:43,140 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:43,140 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E91EC4CF448DBA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:02:43,140 INFO    wandb-upload_1:2119611 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 19:02:44,546 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:02:44,676 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:02:45,302 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310
2024-08-06 19:02:45,302 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5179', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:45,302 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E91EC550E29344</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:02:45,559 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361
2024-08-06 19:02:45,559 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:45,559 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E91EC55F501953</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:02:49,158 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 19:02:49,159 DEBUG   SenderThread:2119611 [sender.py:send():382] send: metric
2024-08-06 19:02:49,159 DEBUG   SenderThread:2119611 [sender.py:send():382] send: metric
2024-08-06 19:02:49,159 DEBUG   SenderThread:2119611 [sender.py:send():382] send: metric
2024-08-06 19:02:49,159 DEBUG   SenderThread:2119611 [sender.py:send():382] send: metric
2024-08-06 19:02:49,159 DEBUG   SenderThread:2119611 [sender.py:send():382] send: history
2024-08-06 19:02:49,159 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: summary_record
2024-08-06 19:02:49,160 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:02:49,547 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/wandb-summary.json
2024-08-06 19:02:49,822 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310
2024-08-06 19:02:49,822 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5179', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:49,822 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E91EC65E3D6187</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:02:50,160 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:02:50,398 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361
2024-08-06 19:02:50,398 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:50,398 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E91EC67BB16E5B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:02:50,547 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:02:51,547 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:02:55,161 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:02:56,396 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 19:02:56,396 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: stop_status
2024-08-06 19:02:56,397 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 19:02:56,498 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 19:02:56,636 DEBUG   SenderThread:2119611 [sender.py:send():382] send: history
2024-08-06 19:02:56,636 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: summary_record
2024-08-06 19:02:56,637 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:02:57,548 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:02:57,548 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/wandb-summary.json
2024-08-06 19:02:58,548 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:02:59,048 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361
2024-08-06 19:02:59,048 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:59,048 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E91EC8835DD991</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:02:59,302 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310
2024-08-06 19:02:59,302 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5179', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:02:59,302 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E91EC893458BCA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:03:00,637 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:03:03,582 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 19:03:03,583 DEBUG   SenderThread:2119611 [sender.py:send():382] send: history
2024-08-06 19:03:03,583 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: summary_record
2024-08-06 19:03:03,583 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:03:04,550 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/wandb-summary.json
2024-08-06 19:03:05,550 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:06,550 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:06,584 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:03:10,913 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 19:03:10,914 DEBUG   SenderThread:2119611 [sender.py:send():382] send: history
2024-08-06 19:03:10,914 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: summary_record
2024-08-06 19:03:10,915 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:03:11,397 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 19:03:11,397 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 19:03:11,397 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: stop_status
2024-08-06 19:03:11,551 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/wandb-summary.json
2024-08-06 19:03:12,552 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:12,582 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:03:13,552 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:13,552 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/config.yaml
2024-08-06 19:03:17,233 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=74f922d15ec38f9d791869d7981269b179334cbed7e8088e8aef60521c92a310
2024-08-06 19:03:17,233 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5179', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:03:17,233 ERROR   wandb-upload_0:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E91ECCC021AB95</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:03:17,646 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:03:18,036 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T110241Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=db066b3f096ebfb6d1b0cb7d7cbd17c1653c2483e230f27ec08a9a6f15fe8361
2024-08-06 19:03:18,036 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:03:18,036 ERROR   wandb-upload_1:2119611 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/e3axglmp/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/e3axglmp/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E91ECCEF4B8878</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:03:18,226 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 19:03:18,226 DEBUG   SenderThread:2119611 [sender.py:send():382] send: history
2024-08-06 19:03:18,227 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: summary_record
2024-08-06 19:03:18,227 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:03:18,553 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/wandb-summary.json
2024-08-06 19:03:19,553 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:20,554 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:23,227 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:03:25,423 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 19:03:25,423 DEBUG   SenderThread:2119611 [sender.py:send():382] send: history
2024-08-06 19:03:25,423 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: summary_record
2024-08-06 19:03:25,424 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:03:25,555 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/wandb-summary.json
2024-08-06 19:03:26,396 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 19:03:26,397 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: stop_status
2024-08-06 19:03:26,397 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 19:03:27,555 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:28,555 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:28,580 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:03:32,686 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 19:03:32,687 DEBUG   SenderThread:2119611 [sender.py:send():382] send: history
2024-08-06 19:03:32,687 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: summary_record
2024-08-06 19:03:32,687 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:03:33,556 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:33,556 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/wandb-summary.json
2024-08-06 19:03:33,688 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:03:34,557 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:38,688 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:03:39,686 DEBUG   SystemMonitor:2119611 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-06 19:03:39,687 DEBUG   SenderThread:2119611 [sender.py:send():382] send: stats
2024-08-06 19:03:39,904 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 19:03:39,905 DEBUG   SenderThread:2119611 [sender.py:send():382] send: history
2024-08-06 19:03:39,905 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: summary_record
2024-08-06 19:03:39,905 INFO    SenderThread:2119611 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:03:40,558 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/wandb-summary.json
2024-08-06 19:03:41,397 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 19:03:41,397 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 19:03:41,397 DEBUG   SenderThread:2119611 [sender.py:send_request():409] send_request: stop_status
2024-08-06 19:03:41,558 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:42,558 INFO    Thread-12 :2119611 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_190239-e3axglmp/files/output.log
2024-08-06 19:03:44,581 DEBUG   HandlerThread:2119611 [handler.py:handle_request():146] handle_request: status_report
